### DISCUSSION TITLE
怎么配置都不通啊，求抄作业

### DISCUSSION THREAD

**Discussion by gmvp3** [XTLS/Xray-core#970] [General]
vless+tcp+xtls nginx
求这2个配置文件，感激~~~

**Comment by gmvp3** [comment]
vless+tcp+xtls 已经配置好了在443
怎么回落80到nginx，求nginx的配置
谢谢啦

**Reply by zanxj** [reply]
Is it convenient to paste your xray configuration ?

**Reply by gmvp3** [reply]
I have used one click script deploy xay，it is OK.
Thank you~~

**Reply by YunofHD** [reply]
> I have used one click script deploy xay，it is OK. Thank you~~

求个一键脚本地址，thx~
